public/locales/ar/gamepage.json

```diff
@@ -93,6 +93,12 @@
         "moving": "قيد النقل",
         "repairing": "قيد الإصلاح"
     },
+    "gamescore": {
+        "header": "Game Scores",
+        "igdb": "IGDB",
+        "metacritic": "MetaCritic",
```

Comment:
Maybe these texts does not need translations, just like Discord, Patreon, for instance.